lib/suite.js

```diff
@@ -24,10 +24,10 @@ exports = module.exports = Suite;
 /**
  * Create a new `Suite` with the given `title` and parent `Suite`.
  *
- * @memberof Mocha
  * @public
  * @param {Suite} parent
  * @param {string} title
+ * @constructs Suite
```

Comment:
Move to L27.5 (i.e., after `@public`)

Reply:
we have not defined the order in which tags appear in docstrings, so I'm going to decline to do this unless we have some way to lint and automate this.